[2023-01-16T08:51:28.673+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2023-01-16T08:51:28.688+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2023-01-16T08:51:28.689+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-16T08:51:28.689+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-01-16T08:51:28.689+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-16T08:51:28.958+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): second_task> on 2021-10-31 00:00:00+00:00
[2023-01-16T08:51:28.962+0000] {standard_task_runner.py:55} INFO - Started process 3108 to run task
[2023-01-16T08:51:28.966+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-31T00:00:00+00:00', '--job-id', '283', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpue1lx25s']
[2023-01-16T08:51:28.969+0000] {standard_task_runner.py:83} INFO - Job 283: Subtask second_task
[2023-01-16T08:51:29.272+0000] {task_command.py:389} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [running]> on host c7f1bfaaf748
[2023-01-16T08:51:29.840+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-31T00:00:00+00:00
[2023-01-16T08:51:29.842+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-01-16T08:51:29.843+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2023-01-16T08:51:29.853+0000] {subprocess.py:86} INFO - Output:
[2023-01-16T08:51:29.855+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2023-01-16T08:51:29.856+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-01-16T08:51:30.061+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211031T000000, start_date=20230116T085128, end_date=20230116T085130
[2023-01-16T08:51:30.343+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-16T08:51:30.370+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
